latest/docs-ref-conceptual/install-azure-cli.md

```diff
@@ -26,6 +26,34 @@ For information about the latest release, see the [release notes](release-notes-
 
 ## <a name="macOS"/>Install on macOS
 
+On macOS, you are able to install either with [Homebrew](https://brew.sh/) or manually.
+
+### Install with Homebrew
+
+1. If you don't have it already, install Homebrew by following the [Homebrew installation instructions](https://docs.brew.sh/Installation.html).
+
+2. Update your local Homebrew repositories.
+
+   ```bash
+   brew update
+   ```
+
+3. Install the `azure-cli` package.
+
+  ```bash
+  brew install azure-cli
+  ```
+
+> [!NOTE]
+> If you previously installed the Azure CLI 1.0 with Homebrew, then instead of installing
+> the package, the new version can be installed through the regular package upgrade process.
```

Comment:
Not sure about this.
What I do know is Azure CLI 1.0 is now at `azure-cli@1` instead of `azure-cli`.

Reply:
From what I recall, the `@` syntax is for version-locking/vendoring. So if somebody wanted to install the CLI 1.0 they would install `azure-cli@1` now, rather than `azure-cli`, which should be updating to new versions as appropriate.

Reply:
Yeah I think we have the same understanding.

> the new version can be installed through the regular package upgrade process.

The above makes it sound like if I had `azure-cli` that was 1.0 then do `brew update/upgrade`, I'll a newer version of 1.0 whereas in-fact I'd get 2.0 now.

Up to you if you think the wording is okay though.

Reply:
Good call, I'll fix the wording and merge.